WallPaperDev/WallPaperDev/Models/CoreDataStack.swift

```diff
@@ -15,7 +15,7 @@ final class CoreDataStack {
     return self.persistentContainer.newBackgroundContext()
```

Comment:
QQ on the file above `WallPaperDev/WallPaperDev.xcworkspace/xcuserdata/jackson_ho.xcuserdatad/UserInterfaceState.xcuserstate` what exactly is this userInterfaceState?